packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -98,6 +104,10 @@ export class CanvasManager implements CanvasManagerInterface {
     this.locked = false;
   }
 
+  public snapshot(canvasElement?: HTMLCanvasElement) {
+    this.manualSnapshot(canvasElement);
+  }
+
```

Comment:
Is this still needed? Seems like we could make/rename `manualSnapshot` to be the public method